src/Footer.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+let footerProp = [
+  "123 Fake Street, London, E1 4UD",
+  "hello@fakehotel.com",
+  "0123 456789",
+];
```

Comment:
This doesnt appear to be being used as a prop, did you mean to pass it as a prop from App?

If not, best not to end a variable in the word `prop`, as thats a well known word for props in react 😄

Reply:
Thanks for pointing that out, I haven't noticed that. :)
The file was created on the original template and I think it will be used later on at the stretch exercises.